pkg/liveshare/port_forwarder.go

```diff
@@ -16,6 +16,33 @@ type portForwardingSession interface {
 	KeepAlive(string)
 }
 
+type ReadWriteHalfCloser interface {
```

Comment:
Open to suggestions for naming here 😅

Reply:
Does the interface need to be public?

Reply:
It doesn't technically need to be I don't think - the reason that I made it public is that it's exposed in the signature of `Forward` as well as `NewReadWriteHalfCloser` but nothing outside this file uses the name